.github/workflows/provider-test.yml

```diff
@@ -42,6 +42,9 @@ jobs:
         run: |
           echo "ARM_OIDC_TOKEN=$(curl -H "Accept: application/json; api-version=2.0" -H "Authorization: Bearer ${ACTIONS_ID_TOKEN_REQUEST_TOKEN}" -H "Content-Type: application/json" -G --data-urlencode "audience=api://AzureADTokenExchange" "${ACTIONS_ID_TOKEN_REQUEST_URL}" | jq -r '.value')"  >>${GITHUB_ENV}
 
+      - name: Set OIDC Token File Path
+        run: echo "${ARM_OIDC_TOKEN}" >"${TMPDIR}/oidc-token.jwt"
```

Comment:
we'll want to move this down two steps, so this is only output if the other two are successful?